tests/parametric/test_otel_env_vars.py

```diff
@@ -0,0 +1,220 @@
+import pytest
+from utils import missing_feature, context, scenarios
+
+
+@scenarios.parametric
+class Test_Otel_Env_Vars:
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
```

Comment:
I think this line is irrelevant due to the `manifests/nodejs.yml` entry
```suggestion
    @missing_feature(context.library < "nodejs@5.11.0", reason="Implemented in v5.11.0, v4.35.0, &v3.56.0")
```

Reply:
I think we want to leave it in there for best practice just to remember which release it was implemented for future reference